RFC-010-core-strategy/RFC-010-core-strategy.md

```diff
@@ -0,0 +1,131 @@
+# RFC010 - Core Strategy
+
+## Proposed by
+
+Felicity Brand, Carrie Crowe
+
+## Current status
+
+See the status indicated on the pull request description.
+
+
+## Proposal overview
+
+This RFC is to gain consensus on several key areas of The Good Docs Project as it stands in 2022. This touches on our vision, audience, content and how we deliver value. 
+The information is contained in the Core Strategy document (core-strategy.md).
+
+The Core Strategy document seeks to outline the fundamentals of the project, answering the following questions:
+
+* What is The Good Docs Project?
+* What do we do?
+* Who is our audience?
+* What content do we offer?
+
+This can then drive our content strategy, and work will follow in future RFCs to answer deeper questions about content substance, workflow, structure and governance.
+
+
+## Motivation
+
+There are several benefits to articulating where we are, how we can best support our users and community members, and where we’d like to grow in future.
+
+Documenting our core strategy means that:
+
+**As a project**, we can: 
+
+* Have a touchstone or “North Star” to guide decisions about content produced by the project, as well as communications about the project.
+* Reinforce the foundation that is already underlying our current initiatives.
+* Adapt easily to support our audience. As we grow, our audience will grow and their needs might change. 
+
+**As a community**, we can:
+
+* Hold a shared idea of what we’re about—so that we’re on the same page. 
+* Present a unified idea, and use consistent language, when talking about the identity of our project to others.
+
+**As content producers**, we can: 
+
+* Address content challenges, prioritize them accordingly and form solutions. 
+* Tie each piece of content that we create back to a community or project goal. 
+
+
+## Proposal
+
+That we accept the following related file:
+
+* core-strategy.md

```

Comment:
When this document was originally created, Felicity and I had a  discussion about this document as a whole living in the collected 'handbook' (alongside the other Content Strategy docs to come)  which would likely be on the website. 

There are contents of this document that I would like to see on  the 'Learn More' page under 'About The Good Docs Project'. I think this would be a great place to have the Core Strategy Statement (The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond) as well as the Who we are, What we do section which could update the 'Goals of the Good Docs Project' section on our website.

Reply:
I've updated the RFC document with these publish points in the 'Implementation' section.